data_steward/analytics/cdr_ops/rt_cdr_qc/cdr_deid_qa_report1_generalization_rule.py

```diff
@@ -26,13 +27,16 @@
 project_id = ""
 com_cdr = ""
 deid_cdr=""
-# deid_base_cdr=""
 pipeline=""
 run_as=""
-# -
 
-# df will have a summary in the end
-df = pd.DataFrame(columns = ['query', 'result']) 
+# +
+# Parameters
+#project_id = "aou-res-curation-prod"
```

Comment:
This parameter should be removed, we do not want our dataset and project names in the git history.